doc/images.md

```diff
@@ -148,6 +148,24 @@ A `SpriteBatchComponent` is also available for your convenience.
 
 See example [here](https://github.com/flame-engine/flame/tree/master/doc/examples/sprite_batch).
 
+## Composition
+
+In some cases you may want to merge multiple images into a single image, this is called [Compositing](https://en.wikipedia.org/wiki/Compositing). This can be useful for example when working with the [SpriteBatch](#spritebatch) API to optimize your drawing calls. 
```

Comment:
```suggestion
In some cases you may want to merge multiple images into a single image; this is called [Compositing](https://en.wikipedia.org/wiki/Compositing). This can be useful for example when working with the [SpriteBatch](#spritebatch) API to optimize your drawing calls. 
```